classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
```

Comment:
- [ ] In implementing  `has-many / belongs-to` relationships between classroom and Student, kindly add an attribute reader to access `students`

Reply:
well resolved! ✔️